molecule/kraft.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
and how can we use this if we want to run some scenarios in zookeeper mode and others in kraft mode on jenkins?

Reply:
We generally don't use any such feature in molecule where you can run multiple scenarios in single molecule command. 
Running multiple scenarios is only possible via Jenkins as of now, where we use a shell script and trigger multiple VM's and run different scenarios.
We have 2 options where one set run with kraft and another set run with zookeeper, 
1.) Run 2 different jenkins job. 
2.) We can modify Jenkins input variable where we pick 2 textboxes, one for scenarios need to run with ZK and another one where we can run with Kraft. An empty input might mean nop scenarios in this part5icula mode.

Reply:
2nd method sounds good to me